packages/upload-api/src/types/blob.ts

```diff
@@ -29,6 +33,16 @@ export interface AllocationsStorage {
     space: DID,
     options?: ListOptions
   ) => Promise<Result<ListResponse<BlobListItem>, Failure>>
+  /** Removes an item from the table but fails if the item does not exist. */
+  remove: (
+    space: DID,
+    digest: BlobMultihash
+  ) => Promise<Result<BlobRemoveSuccess, Failure>>
```

Comment:
Not Found should be success with size zero. See https://github.com/w3s-project/specs/blob/main/w3-blob.md#remove-blob-size

This would be a general failure, like unexpected response from dynamo/store

Reply:
Ah, ok, can you please change the comment above then?